### ISSUE TITLE
Display Date & Time on the opposite sides of timeline panel

### ISSUE THREAD

**Issue by rahil051** [rpocklin/angular-timeline#30]
@rpocklin 
Just like in this bootstrap demo
http://bootsnipp.com/snippets/featured/timeline-dotted
does this module supports date & time display opposite of their respective panel?
So for example timeline panel on the left side of the timeline badge and the date & time component on the right side.
And if it doesn't support the feature, is it planned to be in some future release ?

**Comment by rpocklin** [comment]
I get what you are after - same as #22 looks like it's a wanted feature :)
Should be easy just tweaking some CSS - can you do a PR?

**Comment by rahil051** [comment]
Yeah, I tweaked a little bit of CSS, customized pull-left and pull-right classes by applying even odd logic, i.e.

```
ng-class="{ 'pull-left': $index%2==0, 'pull-right': $index%2!==0 }"
// where $index comes from ng-repeat directive
```

to get it done.
